spec/api/scans_spec.rb

```diff
@@ -23,7 +23,7 @@
   #   labwares = [{position: location.coordinates.first.position, barcode: new_labware.first.barcode},
```

Comment:
I'm guessing these comments can go?

Reply:
Yes - it doesnt seem like using `{ labwares: [] }` in the api/scans params is supported anymore, only `{ labware_barcodes: [] }`